src/random/system.cr

```diff
@@ -0,0 +1,17 @@
+require "sys/random"
+
+# Generates random numbers from a secure source of the system.
+#
+# For example `arc4random` is used on OpenBSD, whereas on Linux it uses
+# `getrandom` (if the kernel supports it) and fallbacks on reading from
+# `/dev/urandom` on UNIX systems.
+class Random::System
+  include Random
```

Comment:
You can use `Random::System` instead of `Random::System.new`. It can act as an instance of `Random` but without a needless object on the heap.
https://github.com/crystal-lang/crystal/pull/3434#discussion_r83738923